pkg/activator/net/throttler_test.go

```diff
@@ -211,6 +211,9 @@ func TestThrottlerWithError(t *testing.T) {
 	}} {
 		t.Run(tc.name, func(t *testing.T) {
 			ctx, cancel, _ := rtesting.SetupFakeContextWithCancel(t)
+			defer func() {
+				cancel()
+			}()
```

Comment:
If we have only cancel function to be called in defer
Can we do
`defer cancel()` ?